decavision/model_training/tfrecords_image_classifier.py

```diff
@@ -197,6 +197,11 @@ def _create_model(self, activation, hidden_size, dropout, l2_lambda):
             base_model = EfficientNetB5(weights='imagenet', include_top=False,
                                         input_shape=(*self.target_size, 3))
             base_model_last_block = 417  # last block 559, two blocks 417
+            
+        elif self.transfer_model == 'B7':
+            base_model = EfficientNetB7(weights='imagenet', include_top=False,
+                                        input_shape=(*self.target_size, 3))
+            base_model_last_block = None  # all layers trainable
```

Comment:
Why are you not specifying the last two blocks?

Reply:
Whenever I used B7, I trained all layers as it gave better results, so I set it to `None`.

Reply:
Interesting! How did the speed compare with just unfreezing the last blocks?

Reply:
It was marginal from my initial experiments, both took some time to train!